Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.swift

```diff
@@ -0,0 +1,520 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2021 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+import UIKit
+
+
+public protocol BPKHorizontalNavigationSize {
+    static var fontStyle: BPKFontStyle { get }
+
+    static var verticalSpacing: CGFloat { get }
+}
```

Comment:
There's no point in allocating an instance here though. Seems cleaner to use static since we are after the type level difference and a couple of static properties